nf_core/pipeline-template/conf/test.config

```diff
@@ -31,4 +31,10 @@ params {
     // Genome references
     genome = 'R64-1-1'
     {%- endif %}
+
+    {% if downstream_samplesheets -%}
+    // Downstream samplesheets
+    generate_downstream_samplesheets = true
+    generate_pipeline_samplesheets   = 'rnaseq'
```

Comment:
do we need to have that as a default?

Reply:
It think we need to add more TODO statemetns next to this, I'm in two minds whether we should put an example or empty.

The whole thing might not be _immediately_ obvious how to adapt without the example. For example I would actually extend the example to comma separated list as this is supported.

Reply:
@jfy133 I like that idea. But we should be carefull not to make it to complex.